### PR TITLE
Add Memory

### PR DESCRIPTION
**Pull request by RadValentin**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

Note: I'm not an employee at Memory currently but I did work for them in 2018, adding them to this list since it was a great remote experience and I'd like to help them out 👍

### CODE REVIEW

File: company-profiles/memory.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Memory
+
+## Company blurb
+
+Memory is not your average Norwegian company. We work tirelessly because we have a need build something bigger than ourselves. To create.
```
Comment (nylen) [inline company-profiles/memory.md]: Typo: "because we have a need **to** build"

File: company-profiles/memory.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Company size
+
+We’re a team of 45 people.
+
+## Remote status
+
+Memory is global company, but we have our roots in Norway. Our team is diverse and represents 20 countries. We’re also big believers in remote work.
```
Comment (nylen) [inline company-profiles/memory.md]: Typo: "Memory is **a** global company"

Comment (RadValentin) [reply company-profiles/memory.md]: Fixed in https://github.com/remoteintech/remote-jobs/pull/618/commits/cc608f585280e66454b2039a1555db4e8cbe4df0

File: company-profiles/memory.md (hunk 1 above)
Comment (RadValentin) [reply company-profiles/memory.md]: Fixed in https://github.com/remoteintech/remote-jobs/pull/618/commits/cc608f585280e66454b2039a1555db4e8cbe4df0

**Comment by dougaitken** [comment]
Not really much explaining what the company are building, otherwise looks good. 

If you're happy with this, we can merge @RadValentin otherwise, please do add some details about the products Memory are building.

**Comment by RadValentin** [comment]
That's a fair point, I'll add more info and ping you for review. Thanks!

**Comment by RadValentin** [comment]
@dougaitken I've updated the blurb so it gives more info on what Memory is about as a company and what they're building. Please take a look, thanks!

**Pull request merged**